src/equations/numerical_fluxes.jl

```diff
@@ -280,7 +280,34 @@ For the derivation of the matrix dissipation operator, see:
   [DOI: 10.1016/j.jcp.2016.12.006](https://doi.org/10.1016/j.jcp.2016.12.006).
 """
 struct DissipationMatrixWintersEtal end
-@inline avg(u_ll, u_rr) = 0.5f0 * (u_ll + u_rr)
+
+@inline function (dissipation::DissipationMatrixWintersEtal)(u_ll, u_rr,
+                                                             orientation::Integer,
+                                                             equations::AbstractEquations{1})
+    return dissipation(u_ll, u_rr, SVector(1), equations)
+end
+
+@inline function (dissipation::DissipationMatrixWintersEtal)(u_ll, u_rr,
+                                                             orientation::Integer,
+                                                             equations::AbstractEquations{2})
+    if orientation == 1
+        return dissipation(u_ll, u_rr, SVector(1, 0), equations)
+    elseif orientation == 2
```

Comment:
```suggestion
    else # orientation == 2
```

Otherwise, we get a type instability since `nothing` is returned depending on the value of `orientation`.